cmd/minishift/cmd/start.go

```diff
@@ -302,17 +303,23 @@ func initSubscriptionManagerFlags() {
 }
 
 // clusterUp downloads and installs the oc binary in order to run 'cluster up'
-func clusterUp(config *cluster.MachineConfig) {
+func clusterUp(cfg *cluster.MachineConfig) {
 	oc := cache.Oc{
```

Comment:
Not separating the `oc` download and caching (oc installation) in this PR as found it need heavy refactoring of `start_test.go` as we can use `instanceConfig.Config.OcPath` here to replace `cmdName` which make `clusterUp()` function to not accept any argument after extracting oc installation implementation in separate function.